### PR TITLE
nixos-remote: add --skip-disko flag

### PR DESCRIPTION
**Pull request by Mic92**
Cherry-picked from https://github.com/numtide/nixos-anywhere/pull/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ Options:
   do not reboot after installation, allowing further customization of the target installation.
 * --kexec url
   use another kexec tarball to bootstrap NixOS
+* --skip-disko
```
Comment (Mic92) [inline README.md:67]:
```suggestion
* --skip-partitioning
```

File: src/nixos-anywhere.sh [hunk 2]
```diff
@@ -293,8 +298,12 @@ for path in "${!disk_encryption_keys[@]}"; do
   ssh_ "umask 077; cat > $path" <"${disk_encryption_keys[$path]}"
 done
 
-nix_copy --to "ssh://$ssh_connection" "$disko_script"
-ssh_ "$disko_script"
+if [[ ${skip_disko} == "y" ]]; then
+  echo "Skipping disko (partitioning)."
```
Comment (Mic92) [inline src/nixos-anywhere.sh:302]: This should actually mount instead of partitioning.

**Comment by Mic92** [comment]
We actually would need mount support first before tackling this.

**Comment by pogobanane** [comment]
another idea is to have a flag for phases: `kexec, reformat, install, reboot`

**Comment by SuperSandro2000** [comment]
I am also interested in this, as reformatting the root disk is a bit tricky for my usecase.

**Comment by Mic92** [comment]
We no have phases as an alternative.

**Pull request closed without merging**